utils/utils.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2019-2020 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+#ifndef UTILS_H_JKZACOAY
+#define UTILS_H_JKZACOAY
+
+#include <stdint.h>
+#define HOST "tangle-accel.puyuma.org"
+#define PORT "443"
+#define API "transaction/"
+#define SSL_SEED "nonce"
+#define REQ_BODY                                                           \
+  "{\"value\": 0, \"tag\": \"POWEREDBYTANGLEACCELERATOR9\", \"message\": " \
+  "\"%s\", \"address\":\"%s\"}\r\n\r\n"
+#define ADDRESS                                                                \
+  "POWEREDBYTANGLEACCELERATOR999999999999999999999999999999999999999999999999" \
+  "999999A"
+#define ADDR_LEN 81
+#define ADDR_LOG_PATH "addr_log.log"
+
+/**
+ * @brief generate trytes from random
+ *
+ * @param len length of output
+ * @param out output array
+ */
+void gen_rand_trytes(uint16_t len, char *out);
+
+/**
+ * @brief log address into addr_log.log
+ *
+ * @param next_addr address to log
+ *
+ * @return log success 0 else -1
+ * @see gen_trytes()
+ */
+int log_address(char *next_addr);
+
+/**
+ * @brief create message to send to tangle accerlator
+ *
+ * @param tryte_msg byte message which has convert to tryte
+ * @param addr IOTA next address
+ * @param req_body[1024] message array
+ */
+void gen_tryte_msg(const char *tryte_msg, const char *addr, char req_body[1024]);
+
+/**
+ * @brief send message vi http protocal
+ *
+ * @param host http host
+ * @param port http port
+ * @param api http api
+ * @param msg message to send
+ * @param msg_len length of message array
+ *
+ * @return http status
+ * @see http_retcode_t
+ */
+int send_https_msg(char const *const host, char const *const port, char const *const api, const char *msg,
+                   const int msg_len);
+#endif /* end of include guard: UTILS_H_JKZACOAY */
```

Comment:
What is `UTILS_H_JKZACOAY`?

Reply:
This is automatically generate by vim snippets.